xml/System.Security.Cryptography/PbeEncryptionAlgorithm.xml

```diff
@@ -40,7 +40,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that encryption be performed with the AES-128 algorithm in CBC mode with PKCS#7 padding.</summary>
```

Comment:
Would it read better like the following?

```suggestion
        <summary>Indicates the encryption should be performed with the AES-128 algorithm in CBC mode with PKCS#7 padding.</summary>
```

Reply:
I could go either way.  Right now we don't have anything that would return one of these values (other than asking PbeParameters what you had already saved), but we theoretically could in the future, so I was having voice issues.

Yours definitely reads more like a sentence someone would say, so let's go with your way :smile:.